packages/playwright-core/src/server/browserContext.ts

```diff
@@ -276,6 +276,15 @@ export abstract class BrowserContext extends SdkObject {
     return await this.doGetCookies(urls as string[]);
   }
 
+  async removeCookies(cookieNames: string | string[]): Promise<void> {
+    const setCookies = await this.cookies();
+
+    const newCookiesSet = setCookies.filter(c => !cookieNames.includes(c.name));
```

Comment:
It seems like `context.removeCookies('long-cookie-name')` will remove cookies with names `long`, `cookie` and `name`.